src/modules/dashboard/components/MotionDomainSelect/MotionDomainSelect.tsx

```diff
@@ -46,8 +48,10 @@ const MotionDomainSelect = ({
        */
       let displayLabel =
         parseInt(option?.value || `${ROOT_DOMAIN_ID}`, 10) === ROOT_DOMAIN_ID
-          ? `${formatMessage(MSG.createDomain)} ${label}`
-          : `${formatMessage(MSG.createDomain)} ${formatMessage({
+          ? `${formatMessage(dropdownLabel || MSG.createDomain)} ${label}`
+          : `${formatMessage(
+              dropdownLabel || MSG.createDomain,
+            )} ${formatMessage({
```

Comment:
Modifying component to accept a custom label. If I didn't do this, I feel like I would basically be copying this component into a new component with a different name for use in this form.